library/VTKExtensions/Rendering/vtkF3DRendererWithColoring.h

```diff
@@ -248,8 +248,9 @@ class vtkF3DRendererWithColoring : public vtkF3DRenderer
 
   /**
    * Convert a component index into a string
+   * Change this method to a non-static method to access a member variable ArrayForColoring
```

Comment:
We do not track changes in the comment, but improving the comment to describe the current behavior would be nice :)

Reply:
changed comment.